src/Smalot/PdfParser/Element/ElementString.php

```diff
@@ -39,30 +39,20 @@
  */
 class ElementString extends Element
 {
-    /**
-     * @param string $value
-     */
     public function __construct($value)
```

Comment:
```suggestion
    public function __construct(string $value)
```

Reply:
This one does not work. Sometimes objects are passed in there. The `ElementDate` extends `ElementString` and in the constructor calls the parent constructor with the same argument as it gets. Thus `ElementString` sometimes gets a `DateTime` object (and not string) as the `$value` parameter in the constructor.